experimenter/experimenter/targeting/constants.py

```diff
@@ -38,6 +38,70 @@ def __post_init__(self):
 CORE_ACTIVE_USERS_TARGETING = "'{event}'|eventCountNonZero('Days', 28, 0) >= 21"
 RECENTLY_LOGGED_IN_USERS_TARGETING = "'{event}'|eventCountNonZero('Weeks', 12, 0) >= 1"
 
+# The following indicate whether the user accepted the terms of use in one of our initial phases of experimentation/rollout
+ACCEPTED_TOU_IN_NIMBUS_EXPERIMENT = "'datareporting.policy.dataSubmissionPolicyAcceptedVersion'|preferenceValue == 3"
+# Accepted in 100% on-train rollout in 139 release (after May 27, 6am PT timestamp), or in earlier partial rollout indicated by presence of on-train rollout population value
+ACCEPTED_TOU_IN_ON_TRAIN_ROLLOUT = f"""
+(
+    (
+        'browser.preonboarding.enrolledInOnTrainRollout'|preferenceValue
+        &&
+        'datareporting.policy.dataSubmissionPolicyAcceptedVersion'|preferenceValue == 2
+    )
+    ||
+    (
+        'datareporting.policy.dataSubmissionPolicyNotifiedTime'|preferenceValue >= '1748350800000'
+        ||
+        'browser.preonboarding.onTrainRolloutPopulation'|preferenceValue
+    )
+)
+"""
+ACCEPTED_TOU_V1 = "'termsofuse.acceptedVersion'|preferenceValue == 1"
+ACCEPTED_TOU = f"""
+(
+    {ACCEPTED_TOU_IN_NIMBUS_EXPERIMENT}
+    ||
+    {ACCEPTED_TOU_IN_ON_TRAIN_ROLLOUT}
+    ||
+    {ACCEPTED_TOU_V1}
+)
+"""
+
+# The following indicate whether the user has changed prefs suggesting they prefer not to see ads or ad-like features
+NEW_TAB_AND_HOMEPAGE_NOT_DEFAULT = "(!homePageSettings.isDefault && !newtabSettings.isDefault)"
+SEARCH_SUGGESTIONS_DISABLED = "!'browser.urlbar.suggest.quicksuggest.sponsored'|preferenceValue"
+SPONSORED_TOPSITES_DISABLED = """
```

Comment:
Since we're trying to match Leif's EXCLUDE THESE USERS spec in the gdoc, I think we want this to be "TOPSITES OR_SPONSORED_TOPSITES_DISABLED".